core/tests/test_utils.py

```diff
@@ -74,11 +74,30 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Set the content, status_code and headers to return in subsequent
+        calls to the urlfetch mock.
+
+        Args:
+            content: str. The content to return in subsequent call to the
+            urlfetch mock.
```

Comment:
In Args sections, always indent continuation lines by 4. Ditto below.

Reply:
Done!